src/features/Home/GetStarted/GetStarted.tsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import styles from './GetStarted.module.scss';
+import { Text } from '@deriv/ui';
+export const GetStarted = () => {
+  return (
+    <div className={`${styles.mainPageRow} ${styles.withPattern}`}>
+      <div className={styles.columnContainer}>
+        <Text type='heading-2' bold data-testid='started-header' role='heading'>
```

Comment:
please use `as` prop here, like so: 
```suggestion
<Text as="h2" />
```
thanks :pray:

Reply:
please check all the other usages of `Text` as well.